test/update_dev_build.sql

```diff
@@ -0,0 +1,84 @@
+-- adding  accounts,  for suppliers
+
+INSERT INTO account (type_id, enterprise_id, number, parent, label)
+VALUES (2, 1, 40111002 ,0, "SUPPLIER'S ACCOUNT 1"),
```

Comment:
Could you define the `id` of account manually with a fixed value, because this `id` can be used in others places